src/Surfnet/StepupMiddleware/ManagementBundle/Validator/IdentityProviderConfigurationValidator.php

```diff
@@ -27,22 +27,29 @@ public function validate(array $configuration, $propertyPath)
     {
         Assertion::isArray($configuration, 'invalid configuration format, must be an object', $propertyPath);
 
-        $acceptedProperties = [
+        $requiredProperties = [
```

Comment:
As discussed, use_pdp is optional (and removed from the list) so all params in $acceptedProperties are now required -> hence requiredProperties.